rpm/rpm.spec

```diff
@@ -33,6 +33,7 @@ BuildRequires: libzstd-devel
 # Need rpm-sign for the work around to include old version of so files
 # can be removed after transition
 BuildRequires: rpm-sign
+BuildRequires: debugedit
```

Comment:
Why do you add this to buildrequires?

debugedit isn't required to build rpm, that would also put debugedit and it's dependencies in the build chain.

Reply:
Where does %install call fin-debugedit?

I don't see it.

Reply:
From an OBS build of  such updated `rpm`:
```
[  107s] Warning: legacy /usr/lib/rpm/find-debuginfo.sh invoked
```

Reply:
That's not from %install it's invoked as part of the post build processing of the built packages. In the transitional step the old find-debuginfo processes the new rpm package which doesn't include fin-debuginfo anymore.

Further the warning would be included in every package built which is for most people noise or confusion at worst.

Reply:
So the warning goes away when we update more build tools, that's good.

It's not noise, it's a reminder to us that there's still a place that needs to be updated. Given how much output (and other noise) package builds produce anyway, intentional "deprecation" oneliner can hardly be considered noise, in my rationale at least.